src/components/Todo.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <li class="todo">
+    <input type="checkbox" v-model="instanceStatus">
+
+    <label>
+      ({{ estimatedSize }}) {{ description }}
+      <a href="#" @click.prevent="deleteTodo">[x]</a>
+    </label>
+  </li>
+</template>
+
+<script>
+import { Vue, Component, Prop, p, Watch } from "av-ts"
+
+@Component({
+  name: "todo"
+})
+class Todo extends Vue {
+  @Prop todoId        = p({ type: Number, required: true })
```

Comment:
(maybe)
PersonalIy, don't like this alignment. cause It broke commit log.
But, If you want it. Keep do it.
https://medium.com/@fagnerbrack/why-aligning-statements-will-haunt-you-c7385a3b24d#.5m33lrazs